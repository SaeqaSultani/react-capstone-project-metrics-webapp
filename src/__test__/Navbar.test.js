import React from 'react';
import { Navbar } from 'react-bootstrap';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

it('Check any changes to the component', () => {
  const tree = renderer.create(
    <BrowserRouter>
      <Navbar />
    </BrowserRouter>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
