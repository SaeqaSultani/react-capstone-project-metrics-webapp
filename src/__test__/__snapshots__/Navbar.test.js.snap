// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check any changes to the component 1`] = `
<nav
  className="navbar navbar-expand navbar-light"
/>
`;
